docs/introduction.md

```diff
@@ -7,21 +7,83 @@ analyses. It offers an infrastructure for rapid development of mass
 spectrometry related software. OpenMS is free software available under the
 three clause BSD license and runs under Windows, macOS, and Linux.
 
-OpenMS has a vast variety of pre-built and ready-to-use tools for proteomics
-and metabolomics data analysis (TOPPTools) as well as powerful 1D, 2D and 3D
-visualization (TOPPView).
-
-OpenMS offers analyses for various quantitation protocols, including label-free
-quantitation, SILAC, iTRAQ, TMT, SRM, SWATH, etc.
-
-OpenMS provides built-in algorithms for de-novo identification and database search,
-as well as adapters to other state-of-the art tools like X!Tandem, Mascot,
-OMSSA, etc. It supports easy integration of OpenMS built tools into workflow
-engines like KNIME, Galaxy, WS-Pgrade, and TOPPAS via the TOPPtools concept and
-a unified parameter handling via a 'common tool description' (CTD) scheme.
-
-With pyOpenMS, OpenMS offers Python bindings to a large part of the OpenMS API
-to enable rapid algorithm development. OpenMS supports the Proteomics Standard
-Initiative (PSI) formats for MS data. The main contributors of OpenMS are
-currently the Eberhard-Karls-Universität in Tübingen, the Freie Universität
-Berlin, and the ETH Zürich.
+```{note}
+Note: This introduction is aimed at users new to the field of LC-MS data analysis and will introduce some basics terms
+and concepts. How to handle the data analysis, available data structures, algorithms and more are covered in the various
+subsections of this documentation.
+```
+
+# Background
+
+Proteomics and metabolomics are interdisciplinary research fields that study structure, function, and interaction of
+proteins and metabolites. They employ large-scale experimental techniques that allow acquiring data at the level of
+cellular systems to whole organisms. One of the main analytical method to identify, characterize or quantify proteins
+and metabolites is mass spectrometry (MS) combined with chromatographic separation.
+
```

Comment:
Mass spectrometry combined with chromatographic separation is commonly used to identify, characterize or quantify the amount of proteins and metabolites.